### DISCUSSION TITLE
Testing json length

### DISCUSSION THREAD

**Discussion by FaierAndreea** [thunderclient/thunder-client-support#893] [Q&A] [answered]
Hi! I want to test the length of a array of objects in a json response, I tried adding a Json Query test and I wrote "json.length" but that says is undefined. What is the correct way?
![Screenshot 2022-11-23 174512](https://user-images.githubusercontent.com/49247127/203589233-fb205dc3-3c95-4c7d-8c9a-f74889afc59a.png)

**Comment by rangav** [comment, marked as answer]
You should input `json` in left input
And select `count` dropdown

**Comment by rangav** [comment]
There is another way to do the same using [Function filters](https://github.com/rangav/thunder-client-support/blob/master/docs/filters.md)

You can use `length` filter and use this format  in left input box `json | length` and select `equal` dropdown
